### PR TITLE
Add ID tags for Homepage and Navbar

### LINKED ISSUE #988: Home Page and Nav Bar: Add ID attribute for elements on these pages
**Please make sure the id attributes are updated for the mobile version as well!!**

Please assign the following elements with the listed JSX 'id' attributes.

Please also note, when assigning id attribute, to assign it to the area that encompasses the entire button or element.
Example: not just the text "Next", but the entire button (including Next and arrow symbol)
Screen Shot 2020-07-03 at 6 43 37 PM

Why we need to assign unique IDs: [https://www.optimizesmart.com/event-tracking-in-google-tag-manager-v2-complete-guide/#:~:text=You%20will%20need%20this%20'id,not%20contain%20any%20white%20space.](url)

## Home Page
| #  | Element on page | JSX 'id' attribute |
| --- | --- | --- |
| 1. | Request Help Button | HP_RE |
| 2. | Offer Help Button | HP_OF |
| 3. | "View Community Posting" Button | HP_VHB |

|Desktop |  Mobile |
| --- | --- |
|   |   |

## Nav Bar
**For users _NOT_ signed in:**
| #  | Element on page | JSX 'id' attribute |
| --- | --- | --- |
| 4. | "About Us" Button | NAV_AU |
| 5. | "Feed" Button | NAV_HB |
| 6. | "Login" Button | NAV_SI |
| 7. | "Register" Button | NAV_JN |
| 8. | Feedback Icon | NAV_FDB |

|Desktop |  Mobile |
| --- | --- |
|   |   |

\
**For users signed in:**
_Please note:_ The buttons "About Us" and "Feed" here should have the same 'id' attributes as the above.
| #  | Element on page | JSX 'id' attribute |
| --- | --- | --- |
| 9. | "Add Organisation" Button | NAV_ADO |

|Desktop |  Mobile |
| --- | --- |
|   |   |

### PR DESCRIPTION
**Pull request by jyclam**
### What does this pull request aim to achieve? 💯
Adds id tags to homepage and navbar

closes #988

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -9,8 +10,9 @@ import MenuIcon from "assets/icons/menu.svg";
 import envelope from "assets/icons/envelope.svg";
 import logo from "assets/logo.svg";
 import Logo from "./Logo";
+
```
Comment (idjevm) [inline client/src/components/Header.js:13]: Remove this extra white space.

Comment (idjevm) [reply client/src/components/Header.js:13]: Maybe addthis import somewhere else, as this section of imports are marked with //icons

Comment (jyclam) [reply client/src/components/Header.js:13]: I added it to logically separate the imports

Comment (idjevm) [reply client/src/components/Header.js:13]: Ohh I see it now. Makes sense :)

Comment (jyclam) [reply client/src/components/Header.js:13]: I could add a // Constants comment but I feel like it's unnecessary to explain imports so explicitly

Comment (idjevm) [reply client/src/components/Header.js:13]: Agee.

**Pull request merged**